src/columns/nano-id.ts

```diff
@@ -0,0 +1,24 @@
+import * as glide from "../glide";
+import { nanoid, customAlphabet } from "../../node_modules/nanoid/async";
+
+export default glide
+    .columnNamed("Nano Id")
+    .withCategory("Text")
+    .withDescription(`A tiny (108 bytes), secure, URL-friendly, unique string ID generator for JavaScript`)
+    .withNumberParam("num", "Length")
```

Comment:
The first parameter should be the seed value that determines the id.

Reply:
See previous comment.